cloudflare.install.sh

```diff
@@ -208,11 +208,15 @@ install $SOURCE_DIR/APIKey $INSTALL_DIR/bin/admin/CloudFlare/APIKey
 chmod 0700 $INSTALL_DIR/bin/admin/CloudFlare/APIKey
 echo "mode=simple" > $INSTALL_DIR/bin/admin/CloudFlare/APIKey.conf
 
+# Get PHP Version
+CPANELSUPPORTEDPHPPATH=`ls -al $INSTALL_DIR/3rdparty/bin/php | awk '{print $11}'`
+PHPVERSION=`echo $CPANELSUPPORTEDPHPPATH | rev | cut -d '/' -f 3 | rev`
```

Comment:
Here's a one liner which will get you the PHP version in the format you want:
```PHPVERSION=`/usr/local/cpanel/3rdparty/bin/php --version | head -1 | awk '{print $2}' | sed 's/\.//' | cut -f 1 -d .````